bridge/runbook/validator_runbook.md

```diff
@@ -184,3 +184,35 @@ Test ingress with curl on a remote machine and expect a `200` response:
 ```bash
 $ curl -v {YOUR_BRIDGE_URL}
 ```
+
+### Bridge Node Monitoring
+(This section is still WIP)
+
+* Use `uptime` to check if the node is running.
+
+* A full list of Bridge Node metrics can be found [here](../../crates/sui-bridge/src/metrics.rs). Find descriptions of each metric [here](../../crates/sui-bridge/src/metrics.rs) and we skip them below.
```

Comment:
Should line 191 and 193 be swapped here? Not sure which is clearer so feel free to ignore

Reply:
i actually switched them once lol, cuz i feel `uptime` is more obvious to mention first, but no royal to this change

Reply:
edit: after seeing markdown view, I think it's good as is